### PR TITLE
[W5.11][W14-A3]Heng Yeow

### PR DESCRIPTION
**Pull request by tanhengyeow**
Added SearchCommand to search for names in the addressbook who contain the specified letters.

### CODE REVIEW

File: src/seedu/addressbook/commands/SearchCommand.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                if (person.getName().fullName.indexOf(currLetter) >= 0) {
+                    matchedPersons.add(person);
+                    break;
+                }
+            }
+        }
+        return matchedPersons;
+    }
+
+}
```
Comment (waynekoo) [inline src/seedu/addressbook/commands/SearchCommand.java:69]: Very neatly done! 👍 😄

File: src/seedu/addressbook/commands/SearchCommand.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    /**
+     * Retrieves all persons in the address book whose names contain some of the specified letters.
+     *
+     * @param letters for searching
+     * @return list of persons found
+     */
+    private List<ReadOnlyPerson> getPersonsWithNameContainingAnyLetters(String letters) {
+        final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+
+        // Returns a list of people whose names contains letters specified by user
```
Comment (randypx) [inline src/seedu/addressbook/commands/SearchCommand.java]: This comment is basically a repeat of the outside comments. It is not needed.

File: src/seedu/addressbook/commands/SearchCommand.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        for (ReadOnlyPerson person : addressBook.getAllPersons()) {
+
+            for (int i=0; i<letters.length(); i++) {
```
Comment (kowshik-sundararajan) [inline src/seedu/addressbook/commands/SearchCommand.java]: Do keep in mind the spacing in the expression (ex: should be `i = 0`).

**Review by kowshik-sundararajan — COMMENTED** [review]
Good job! 👍Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**